### ISSUE TITLE
QiDao - Issues

### ISSUE THREAD

**Issue by 0xroll** [delta-hq/l2-lxp-liquidity-reward#229]
hi @royalaid 

as discussed in the previous PR https://github.com/delta-hq/l2-lxp-liquidity-reward/pull/206#issuecomment-2172383628

have you discussed with your team on this?

it makes more sense to be tracking the collateral instead of the debt.

**Comment by royalaid** [comment]
Yep, we have swapped the debt tracking to collateral deposit tracking tracking. PR can be found here. https://github.com/delta-hq/l2-lxp-liquidity-reward/pull/234

**Comment by 0xroll** [comment]
hey @royalaid , just checked the data after merging latest PR

only getting 14 users for block number 7842904 , total sum of tvl doesnt seem close to what defillama is reporting

![image](https://github.com/user-attachments/assets/fccabbe9-93cc-46f1-9d5f-682b3d86b18e)


[qidao_7842904.csv](https://github.com/user-attachments/files/16537632/qidao_7842904.csv)

**Comment by royalaid** [comment]
Just seeing this now, investigating
